README.md

```diff
@@ -122,10 +122,12 @@ In case the patch is applied only on certain version of the package, a version r
 {
   "extra": {
     "patches": {
-      "description for my patch": {
```

Comment:
both of these formats actually work. the second one will dictate the sequence for the patches to be applied for that specific package (thus the usage of array). Although - to be honest - the fact that there's now such high diversification in how one can define their patches - it might actually be pretty confusing for users and sequenced version would probably be preferred in any case :)